### PR TITLE
Document the HoursOpen element. Update the spec to require at least o…

### PR DESCRIPTION
**Pull request by jdmgoogle**
…ne Schedule element per HoursOpen element (i.e., empty hours are not allowed).

### CODE REVIEW

**Comment by jdmgoogle** [comment]
This addresses issue #197.

File: docs/xml/elements/hours_open.rst [hunk 1]
```diff
@@ -1,57 +1,103 @@
...
-| Tag      | Data Type          | Required?  | Repeats? |Description           |Error Handling              |
-|          |                    |            |          |                      |                            |
-+==========+====================+============+==========+======================+============================+
-| Schedule | Schedule           | Optional   | Repeats  |                      |                            |
-+----------+--------------------+------------+----------+----------------------+----------------------------+
++----------+--------------------+------------+----------+-----------------------+-----------------------------------+
+| Tag      | Data Type          | Required?  | Repeats? |Description            |Error Handling                     |
+|          |                    |            |          |                       |                                   |
++==========+====================+============+==========+=======================+===================================+
+| Schedule | Schedule           |**Required**| Repeats  |Defines a block of days| At least one valid `Schedule`_    |
```
Comment (jungshadow) [inline docs/xml/elements/hours_open.rst]: Does the \`Schedule\`_ link work? Shouldn't it be \`HoursOpen.Schedule\`_? The heading could change to just \`Schedule\`_, which will fit better in the sidebar navigation.

File: docs/xml/elements/hours_open.rst [hunk 2]
```diff
@@ -1,57 +1,103 @@
...
+|          |                    |            |          |will be open.          | to be valid. If no valid          |
+|          |                    |            |          |                       | `Schedule`_ is present, the       |
+|          |                    |            |          |                       | implementation is required to     |
+|          |                    |            |          |                       | ignore the ``HoursOpen`` element. |
++----------+--------------------+------------+----------+-----------------------+-----------------------------------+
 
 HoursOpen.Schedule
 ------------------
 
+A sub-portion of the schedule. This describe a range of days, along with one or
```
Comment (pkoms) [inline docs/xml/elements/hours_open.rst]: "describe" should be "describes"

File: docs/xml/elements/hours_open.rst [hunk 3]
```diff
@@ -1,57 +1,103 @@
...
+more set of open and close times for those days, as well as the options
+describing whether or not appointments are necessary or possible.
+
 +---------------------+------------------+-----------+----------+----------------------+----------------------------+
 | Tag                 | Data Type        | Required? | Repeats? |Description           |Error Handling              |
 |                     |                  |           |          |                      |                            |
 +=====================+==================+===========+==========+======================+============================+
-| Hours               | Hours            | Optional  | Repeats  |                      |                            |
+| Hours               | Hours            | Optional  | Repeats  |Blocks of hours in the|If the field is invalid or  |
```
Comment (jungshadow) [inline docs/xml/elements/hours_open.rst]: Could be helpful as an in-page anchor link by using \`Hours\`_ and changing the "Hours" header (same reasons as above).

File: docs/xml/elements/hours_open.rst [hunk 4]
```diff
@@ -1,57 +1,103 @@
...
 
-   <HoursOpen id="hours0000">
-     <Schedule>
-       <Hours>
-         <StartTime>08:30:00-05:00</StartTime>
-	 <EndTime>17:00:00-05:00</EndTime>
-       </Hours>
-     </Schedule>
-   </HoursOpen>
+  <HoursOpen id="hours0001">
```
Comment (pkoms) [inline docs/xml/elements/hours_open.rst]: Shouldn't there be a UTC offset on these hours?

Comment (jungshadow) [inline docs/xml/elements/hours_open.rst]: @pkoms Isn't the -05:00 the offset?

**Comment by jungshadow** [comment]
I have a couple of simple aesthetic/usability/linking concerns, but, other than that, LGTM!

File: docs/xml/elements/hours_open.rst (hunk 4 above)
Comment (pkoms) [inline docs/xml/elements/hours_open.rst]: I'm so used to early vote sites that open and close when the clerk goes to get lunch... yes, -05:00 is the offset.

File: docs/xml/elements/hours_open.rst (hunk 1 above)
Comment (jdmgoogle) [inline docs/xml/elements/hours_open.rst]: Fixed.

File: docs/xml/elements/hours_open.rst (hunk 2 above)
Comment (jdmgoogle) [inline docs/xml/elements/hours_open.rst]: Fixed.

File: docs/xml/elements/hours_open.rst (hunk 3 above)
Comment (jdmgoogle) [inline docs/xml/elements/hours_open.rst]: Fixed.

**Comment by jdmgoogle** [comment]
Comments addresses. PTAL.

File: docs/xml/elements/hours_open.rst [hunk 5]
```diff
@@ -1,57 +1,103 @@
...
+A structured way of describing the days and hours that a place such as a
+:doc:`Office <office>` or :doc:`PollingLocation <polling_location>` is open, or
+that an event such as an :doc:`Election <election>` is happening.
 
-   Document HoursOpen & Schedule
++----------+--------------------+------------+----------+-----------------------+----------------------------------+
+| Tag      | Data Type          | Required?  | Repeats? |Description            |Error Handling                    |
+|          |                    |            |          |                       |                                  |
++==========+====================+============+==========+=======================+==================================+
+| Schedule | Schedule           |**Required**| Repeats  |Defines a block of days|At least one valid `Schedule`_    |
```
Comment (jungshadow) [inline docs/xml/elements/hours_open.rst]: Sorry didn't notice this before. Just need a link for "Schedule" in the Data Type column, as well.

**Comment by jungshadow** [comment]
Again, with fix, LGTM. /cc @pkoms @pstenbjorn @cjerdonek

**Comment by pkoms** [comment]
LGTM.

**Comment by jdmgoogle** [comment]
Thanks. Merging.

**Pull request merged**